plugins/modules/inventory_workflow_manager.py

```diff
@@ -2612,16 +2628,23 @@ def update_interface_detail_of_device(self, device_to_update):
                     self.log("Received API response from 'update_interface_details': {0}".format(str(response)), "DEBUG")
 
                     if response and isinstance(response, dict):
-                        task_id = response.get('response').get('taskId')
+                        response = response.get('response')
+                        if not response:
+                            self.status = "failed"
+                            self.msg = "Interface Updation get failed due to empty response from the update_interface_details API."
```

Comment:
self.msg = "Failed to update the interface because the 'update_interface_details' API returned an empty response."

Reply:
updated.